tests/foreman/cli/test_docker.py

```diff
@@ -1343,6 +1343,128 @@ def test_positive_pull_image(self):
                 hostname=self.docker_host.ip_addr
             )
 
+    @run_only_on('sat')
+    @skip_if_not_set('docker')
+    @tier3
+    def test_positive_container_admin_end_to_end_search(self):
+        """Verify that docker command line can be used against
+        Satellite server to search for container images stored
+        on Satellite instance.
+
+        :id: cefa74e1-e40d-4f47-853b-1268643cea2f
+
+        :steps:
+
+            1. Publish and promote content view with Docker content
+            2. Set "Unauthenticated Pull" option to false
+            3. Try to search for docker images on Satellite
+            4. Use Docker client to login to Satellite docker hub
+            5. Search for docker images
+            6. Use Docker client to log out of Satellite docker hub
+            7. Set "Unauthenticated Pull" option to true
+            8. Search for docker images
+
+        :expectedresults: Client can search for docker images stored
+            on Satellite instance
+
+        :CaseLevel: System
+        """
+        pattern_prefix = gen_string('alpha', 5)
+        docker_upstream_name = 'alpine'
+        registry_name_pattern = ("{}-<%= content_view.label %>"
+                                 "/<%= repository.docker_upstream_name %>").format(
+                pattern_prefix)
+
+        # Satellite setup: create product and add Docker repository;
+        # create content view and add Docker repository;
+        # create lifecycle environment and promote content view to it
+        product = make_product_wait({'organization-id': self.org['id']})
+        repo = _make_docker_repo(product['id'],
+                                 upstream_name=docker_upstream_name)
+        Repository.synchronize({'id': repo['id']})
+        repo = Repository.info({'id': repo['id']})
```

Comment:
`id` is not changed, so do you really need that line? or in case you read the content, maybe you can check that your sync was really successful and necessary number of packages is there?

Reply:
You're right, I removed it.